giraffe/src/components/SimpleTable/PaginationNav/Button/Button.scss.d.ts

```diff
@@ -0,0 +1,27 @@
+// This file is automatically generated.
+// Please do not change this file!
+interface CssExports {
+  'ButtonSpinner': string;
+  'active': string;
+  'cf-button': string;
+  'cf-button--disabled': string;
+  'cf-button--label': string;
+  'cf-button--loading': string;
+  'cf-button-danger': string;
+  'cf-button-default': string;
+  'cf-button-icon': string;
+  'cf-button-lg': string;
+  'cf-button-md': string;
+  'cf-button-primary': string;
+  'cf-button-secondary': string;
+  'cf-button-sm': string;
+  'cf-button-spinner': string;
+  'cf-button-square': string;
+  'cf-button-stretch': string;
+  'cf-button-success': string;
+  'cf-button-tertiary': string;
+  'cf-button-warning': string;
+  'cf-button-xs': string;
+}
+export const cssExports: CssExports;
+export default cssExports;
```

Comment:
This file should not be added. It is auto-generated with `yarn build` when there is a consuming component.

Reply:
`yarn build` throwing errors saying that `./Button.scss` could not be found. However, with all the pieces together `yarn build` does work, so just going to remove this for now.